scheduler/test/cook/test/scheduler/scheduler.clj

```diff
@@ -863,14 +966,79 @@
                        vals
                        (reduce #(+ %1 (count (.getTasksAssigned %2))) 0))))))))
 
+(deftest test-attr-equals-host-placement-constraint-k8s
```

Comment:
Why can this not overlap with the mesos test? E.g., a deftest can call another function like:

(defn do-some-tests [arg1 arg2 arg3]
  (is ....))

(deftest test-foo
  (do-some-tests (make-foo ....) ....))

(deftest test-bar
   (do-some-tests (make-bar ...) ....))